FAQ.md

```diff
@@ -11,52 +11,51 @@ Level 3
 https://yeticold.com/Cold/step1
 
 ### Should I read this FAQ if I am a normal user?
-Probably not. Yeti was designed so that you can simply start at yeticold.com and follow simple instructions to end up with a bitcoin wallet that is appropriate for the amount of bitcoin you are storing. If you get stuck on a step you should watch the videos linked above or reach out on slack:
+Yeti was designed so that you can simply start at yeticold.com and follow simple instructions to end up with a bitcoin wallet that is appropriate for the amount of bitcoin you are storing. If you have documentation questions, this is a good place to start. If you get stuck on a step you should watch the videos linked above or reach out on slack:
 
 https://join.slack.com/t/yeticold/shared_invite/zt-hisfxrra-BZzrYCDnqFv6whxVn~6FQQ
 
 ### How much knowledge of Linux is required?
-None. Installing Ubuntu is the only challenging task and it is no more difficult than installing Windows or Mac OS. Even if you have never installed an OS before it is not difficult and you can get support from our slack channel at:
+None. Installing Ubuntu is the only challenging task and it is no more difficult than installing Windows or Mac OS. Even if you have never installed an OS before it is not difficult. Our guide offers instructions, and the linked videos offer a more comprehensive walkthrough. You can also get support from our slack channel at:
 
 https://join.slack.com/t/yeticold/shared_invite/zt-hisfxrra-BZzrYCDnqFv6whxVn~6FQQ
 
-You can also hire a local computer shop to install Ubuntu for you for about $30 and as long as they don't suspect you will use this laptop for bitcoin it is a reasonably safe option.
+You can also hire a local computer shop to install Ubuntu for you for about $30 and as long as they don't suspect you will use this laptop for bitcoin it is a reasonably safe option for lower levels.
```

Comment:
I think this is a reasonably safe option even for Level 3.

Reply:
Fixed.